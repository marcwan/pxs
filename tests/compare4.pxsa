        ADDVAR          x
        SET             x, 500
        ADDVAR          y
        SET             y, "123"
        COMPARE         x, y
        DEBUGPRINT      :::FLAGS:::
