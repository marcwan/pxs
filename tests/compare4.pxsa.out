DEBUGPRINT COMPARE FLAGS: 0x1
